pages/common/ffmpeg.md

```diff
@@ -16,7 +16,11 @@
 
 - Quickly extract a single frame from a video at time mm:ss and save it as a 128x128 resolution image:
 
-`ffmpeg -ss {{mm:ss}} -i {{video.mp4}} -frames 1 -s {{128x128}} -f image2 {{image.png}}`
+`ffmpeg -ss {{mm:ss.S}} -i {{video.mp4}} -frames 1 -s {{128x128}} -f image2 {{image.png}}`
+
+- Trim a video from a given time (ss) to (t) seconds from that time or just omit -t to trim everything before (ss) seconds:
```

Comment:
The "given time" should be in "mm:ss.S"

Reply:
Like....
"Trim a video from "mm:ss.S" (ss) to (t) seconds [...]" ? .-.